package.json

```diff
@@ -62,11 +62,11 @@
   "dependencies": {
     "@types/vexflow": "^1.2.33",
     "jspdf-yworks": "^2.1.1",
-    "jszip": "3.4.0",
+    "jszip": "^3.4.0",
```

Comment:
this change should also not be a part of this pull request.
if you can simply revert these package.json changes, we can probably do a pull request straight to opensheetmusicdisplay/develop.